lib/travis/client/session.rb

```diff
@@ -17,7 +17,7 @@ module Travis
   module Client
     class Session
       PRIMITIVE   = [nil, false, true]
-      SSL_OPTIONS = { :ca_file => Tools::Assets['cacert.pem'] }
```

Comment:
It's possible. but my idea is using system CAs instead. If you just want to change feature for Enterprise only, I can do that now.